README.md

```diff
@@ -10,14 +10,31 @@ Internet media / content / MIME types.
 
 ## Installation
 
+Install `media-types` with [Spago](https://github.com/purescript/spago):
+
 ```
-bower install purescript-media-types
+spago install media-types
 ```
 
```

Comment:
We usually add a placeholder “quick start” to this section (and open a corresponding issue) — contrib-updater adds the section boilerplate, and then we add the issue manually.

Reply:
A placeholder "quick start" section was added by `contrib-updater`.

Adding a quick start guide is tracked by #12.